docker/op/cert.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I usually prefer to just create a self signed cert on the fly. This one is only valid till Oct 2019, so would this work after that date too?

Reply:
OK, I will make a separate issue for that.

Reply:
yes, sounds fine. For testing its not really an issue.